### DISCUSSION TITLE
License Issue

### DISCUSSION THREAD

**Discussion by Thunder88Mint** [ansys/pyfluent#1642] [Q&A] [answered]
I'm having issues with connecting to my license.  We have the "pro" capability level.  I run the following code:

`import ansys.fluent.core as pyfluent
session = pyfluent.launch_fluent(mode="meshing", precision="double", processor_count=2, show_gui=True)`

And I'm getting the following error message when the gui opens:

> ANSYS LICENSE MANAGER ERROR:Request name cfd_preppost does not exist in the licensing pool.

I looked on our licensing server and the closest license that I could see is 'cfd_preppost_pro'
Let me know if anyone knows how to fix this?

**Comment by dnwillia-work** [comment, marked as answer]
Yes, you can't use pyfluent with a pro level commercial license at this time.  You need premium or above.
